lib/domain/model/alert.dart

```diff
@@ -73,6 +85,10 @@ class Alert {
 
   static Alert fromJson(Map<String, Object?> json) => Alert(
       id: json[AlertFields.id] as int?,
+      status: AlertStatus.values.firstWhere(
+            (status) => status.toString().split('.').last == json[AlertFields.status],
```

Comment:
why do we need to call split here?

Reply:
Yeah because status is going to be something like `AlertStatus.pending`, but we only need the `pending` part.  Didn't find a more clever way to iterate enum values 😓